app/2.0/start/quick-tour.md

```diff
@@ -41,6 +49,11 @@ contain a dash (-)**.
   <iframe frameborder="0" src="samples/custom-element/index.html" width="100%" height="40"></iframe>
 </demo-tabs>
 
+* Try modifying the contents of `this.textContent`. 
```

Comment:
:heart: the prompts. Is it obvious enough that they need to click the plunker buttons to make these changes?